app/controllers/events_controller.rb

```diff
@@ -1,5 +1,5 @@
 class EventsController < ApplicationController
-  before_action :set_event, only: [:show, :edit, :update, :destroy]
```

Comment:
From what I know you have this all actions in event controller and it was OK here. Did you test it in the browser after change? We discussed to remove this code from member controller.